tsconfig.base.jsonc

```diff
@@ -30,6 +30,16 @@
     "skipLibCheck": true, // Skip type checking of third party libraries.
     "noErrorTruncation": false, // Do not truncate error messages.
     "types": [], // Disable automatic loading of `@types/*` packages.
-    "plugins": [{ "name": "@effect/language-service" }]
+    "plugins": [
+      {
+        "name": "@effect/language-service",
+        "namespaceImportPackages": [
+          "effect",
+          "@effect/platform-bun",
```

Comment:
Mattia added support for wildcards here. So `@effect/*` should work afaik. ./cc @mattiamanzati